ui/src/utilities/api.ts

```diff
@@ -2,14 +2,24 @@ import { createApi, PrefectConfig } from '@prefecthq/prefect-ui-library'
 import { createActions } from '@prefecthq/vue-compositions'
 import { InjectionKey } from 'vue'
 import { AdminApi } from '@/services/adminApi'
+import { CsrfTokenApi, setupCsrfInterceptor } from '@/services/csrfTokenApi'
+import { AxiosInstance } from 'axios';
+
+
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export function createPrefectApi(config: PrefectConfig) {
-  const workspaceApi = createApi(config)
+export async function createPrefectApi(config: PrefectConfig) {
+  const csrfTokenApi = new CsrfTokenApi(config)
```

Comment:
You'll probably want to wrap this in `createActions` since its being returned as part of the workspace api. That just creates an object of the instance methods with `this` pre-bound`. That way when we create subscriptions the context isn't lost. 

```suggestion
  const csrfTokenApi = createActions(new CsrfTokenApi(config))
```